src/language-css/printer-postcss.js

```diff
@@ -182,6 +184,8 @@ function genericPrint(path, options, print) {
             !parentNode.raws.semicolon &&
             options.originalText[options.locEnd(node) - 1] !== ";"
           ? ""
+          : options.__isHTMLStyleAttribute && isLastNode
```

Comment:
https://github.com/bschlenk/prettier/blob/6c8f642ec20693d5a23e97e4b36ddfc56d139a09/src/language-css/printer-postcss.js#L43

We already has this util